### PR TITLE
Standardize dollar format

### PR DESCRIPTION
**Pull request by markswinimer**
Used d3-format to add formatting to dollar values. Added comments where format was added to inform future changes.

### CODE REVIEW

File: src/components/file-upload.tsx [hunk 1]
```diff
@@ -155,6 +156,7 @@ export class GenerateTable extends React.Component<GenerateTableProps> {
     } else if (manual) {
       tableRows = Object.keys(manualTable).map((year, key) => {
         const earningValue = manualTable[year]
+        /* Can this be parseable with symbols $, */
```
Comment (thadk) [inline src/components/file-upload.tsx]: Oops, is this a question for your next version or something that should go into the code?
```suggestion
```

File: src/components/file-upload.tsx [hunk 2]
```diff
@@ -134,6 +134,7 @@ export class GenerateTable extends React.Component<GenerateTableProps> {
       var earningsYears = Object.keys(parsedXml);
       tableRows = earningsYears.map((year, i) => {
         const earningValueXML = parsedXml[year]
+        /* Can this be parseable with symbols $, */
```
Comment (thadk) [inline src/components/file-upload.tsx]: See below
```suggestion
```

**Review by thadk — CHANGES_REQUESTED** [review]

File: src/components/file-upload.tsx (hunk 2 above)
Comment (thadk) [reply src/components/file-upload.tsx]: Looks good otherwise! I can commit this suggestion and get this merged in if you agree.

Comment (markswinimer) [reply src/components/file-upload.tsx]: Thanks Thad, that works for me. In the future I'll just capture something like this in notes and/or eventually an issue.

**Review by thadk — APPROVED** [review]

**Pull request merged**